ui/admin/app/controllers/scopes/scope/roles/role/scopes.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Controller, { inject as controller } from '@ember/controller';
+import { action } from '@ember/object';
+import {
+  GRANT_SCOPE_THIS,
+  GRANT_SCOPE_CHILDREN,
+  GRANT_SCOPE_DESCENDANTS,
+} from 'api/models/role';
+
+export default class ScopesScopeRolesRoleScopesController extends Controller {
+  @controller('scopes/scope/roles/index') roles;
+
+  @action
```

Comment:
Should these be actions? It seems like these are just local helper functions.

Reply:
No they shouldn't be actions. Removed.